app/src/main/java/com/example/areumdap/Network/TokenAuthenticator.kt

```
@@ -0,0 +1,90 @@
+
+                // 실패했던 요청을 새 토큰으로 재시도
+                response.request.newBuilder()
+                    .header("Authorization", "Bearer ${newTokens.accessToken}")
+                    .build()
+            } else {
+                // 재발급 실패 → 로그아웃
+                TokenManager.clearAll()
+                null
+            }
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)